core-services/openshift-priv/_whitelist.yaml

```diff
@@ -0,0 +1,36 @@
+whitelist:
+  openshift:
+    - kubernetes
+    - kubernetes-namespace-reservation
```

Comment:
this one is not an upstream fork, but our own repo

Reply:
Thanks. removed.